retroauth-android/src/main/java/com/andretietz/retroauth/AccountAuthenticator.kt

```diff
@@ -125,6 +125,6 @@ class AccountAuthenticator(
   }
 
   companion object {
-    const val KEY_CREDENTIAL_TYPE = "account_credential_type"
+    internal const val KEY_CREDENTIAL_TYPE = "account_credential_type"
```

Comment:
Any reason you changed this? I had a clone of AuthenticatorActivity and I can't access it now because it's internal.